src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -0,0 +1,88 @@
+/* eslint complexity: [2, 25] */
+import {FORMS} from './RemediationConstants';
+
+const FORMNAME_CLASSNAME_MAPPINGS = {
+  [FORMS.IDENTIFY]: {
+    'identify': ['identify', 'primary-auth'],
+    'password': ['identify', 'primary-auth']
```

Comment:
looks like this is not used.

Reply:
This is so that we can add the `primary-auth` classname for when we have password authenticator in response on the identifier screen i.e identifier + password on login screen use case.